planetary-social/nos
# Remove new moderation flow feature flag

## Issues covered
#1646

## Description
Removes the "new moderation flow" feature flag and old moderation flow code.

## How to test
1. Build the code.
2. Navigate to settings screen.
3. Please check that there is no "new moderation flow toggle", and the new moderation flow is the one being displayed.

## Screenshots/Video
| Before | After |
|--------|-------|
| <img src="https://github.com/user-attachments/assets/8b032db5-47db-46d4-828f-cdbd9926f061" alt="Before" width="250"/> | <img src="https://github.com/user-attachments/assets/e4337408-7098-4851-94cc-6b116578cfa5" alt="After" width="250"/> |
